src/Pagination/index.jsx

```diff
@@ -264,7 +266,7 @@ class Pagination extends React.Component {
       buttonLabels, pageCount, icons, variant, size,
     } = this.props;
     const { currentPage } = this.state;
-    const isLastPage = currentPage === pageCount;
+    const isLastPage = (currentPage === pageCount) || (pageCount <= 1);
```

Comment:
Is it necessary to have `isLastPage` be dependent on `pageCount` when both cases are handled independently already in `disabled` below?

I also realized we'll probably want to ensure the `tabIndex={-1}` is kept in sync with whenever `isDisabled` is true.

Perhaps something along the lines of:

```jsx
const isLastPage = currentPage === pageCount;
const isDisabled = isLastPage || pageCount <= 1;

// ...

tabIndex={isDisabled ? -1 : undefined}
disabled={isDisabled}
```

Similar feedback for the previous chevron `IconButton` as well.

Reply:
Ideally no, but not taking it into account allows the next button to be clickable on the default variant state when `pageCount = 0`, because the default `currentPage` is set to 1. So by setting the last page boolean based on strictly the value of `pageCount` as opposed to checking if the `currentPage === pageCount`. 
![Screen Shot 2022-12-23 at 1 01 20 PM](https://user-images.githubusercontent.com/82611798/209385822-6aafc00c-01b6-40f6-95d6-96ed0efb1d65.png)

Reply:
Ahh but looking at your fix, that does fix the issue as well